pkgs/development/node-packages/node-packages.nix

```diff
@@ -84332,426 +84332,6 @@ in
     bypassCache = true;
     reconstructLock = true;
   };
-  "@bitwarden/cli" = nodeEnv.buildNodePackage {
```

Comment:
wont this get regenerated next time ? or the updater ignores everything aliased ?

Reply:
It was already removed from `node-packages.json` in the previous PR at https://github.com/NixOS/nixpkgs/pull/218923/files#diff-9388697b10937594885d71791491e681da1807310617ed9ad03927f9776cb309L6